src/index.ts

```diff
@@ -64,19 +65,27 @@ import {
 	CharacterData,
 	Comment,
 	Document,
-	Element,
+	Element as FontoXPathElement,
```

Comment:
Why this cast?

Reply:
In line 88, xpathExpression should be typed as an Element but not the one in fontoxpath/types/Types. Meanwhile we need to export Element in fontoxpath as Element. So I needed to cast it.